middleman_protocol/tests/utils.py

```diff
@@ -0,0 +1,16 @@
+import assertpy
+
+
+def assertpy_bytes_starts_with(string: bytes, string_to_check: bytes):
+    """
+    Custom function which allows testing if given bytes start with other bytes using assertpy.
+    """
+
+    assert isinstance(string, bytes)
+    assert isinstance(string, bytes)
```

Comment:
Wouldn't it be clearer to call it `byte_array`, `binary_data` or even `data` rather than `string` if it's not supposed to be a string? :)

Or maybe just allow using `str`, because it looks like it should work with it just fine.

Reply:
No no, `str` works with `assertpy` by default, I need to cover only `bytes` here. Names changed to `data` and `starting_data`. :+1:

Reply:
I know you can use `str` with the original method but I just wanted to point out that there's no reason for your function to reject `str` because it can work with it too without any additional code. It would be more versatile and you could use it with both types for consistency if you wanted to.

But anyway, it's just a minor thing. The code as is is fine too.